### PR TITLE
app - Views

### PR DESCRIPTION
**Pull request by frnega**
**In this pull request **
Created a View path same as wireframes.

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
+  <div class="card-body">
+    <h5 class="card-title">Bio</h5>
+    <p class="card-text"><%= @user.bio || '...' %></p>
+  </div>
+</div>
+
+<h1 class="text-center m-5">Posts</h1>
+
+<% if !@user.posts.empty? %>
+  <%= render @user.posts, locals: {truncate: true, show_comments: false} %>
```
Comment (ichala) [inline app/views/users/show.html.erb]: - [ ] as part of the requirements the user page should only show recent posts and not all of them , you can use the method that you have created before `recent_posts` , kindly fix that 🙏

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,11 @@
...
+
+<h1 class="text-center m-5">Posts</h1>
+
+<% if !@user.posts.empty? %>
+  <ul class="col p-0">
+    <%= render @user.posts, locals: {truncate: true} %>
+  </ul>
+<% else %>
+  <p class="text-center m-5">There are no posts yet</p>
+<% end %>
```
Comment (ichala) [inline app/views/posts/index.html.erb]: - [ ] kindly fix the show all posts view where I'm getting this error and make sure it matches the wireframe 🙏 


error: 

image



wireframe: 
image

File: app/views/posts/show.html.erb [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
+  <span class="ms-auto">Comments: <%= @post.comments_counter %> | Likes: <%= @post.likes_counter %></span>
+  <div class="card-body">
+    <h5 class="card-title"><%= @post.title %></h5>
+    <p class="card-text"><%= @post.text || '...' %></p>
+  </div>
+</div>
+
+<ul class="col p-0">
+  <%= render @post.comments %>
+</ul>
```
Comment (ichala) [inline app/views/posts/show.html.erb]: - [ ] kindly make sure that the post page matches the wireframe 🙏 

your design 

image

wireframe 
image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Great Design ✔️ 
- Created Views ✔️ 
- No lInter errors ✔️ 



 ## Required Changes ♻️

- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by frnega** [comment]
made changes to the database added new data to the database
users page
user745
post745
post6

File: app/views/posts/show.html.erb (hunk 3 above)
Comment (danmainah) [reply app/views/posts/show.html.erb]: - [ ] @frnega kindly ensure your design matches the wireframe given as previously requested, your comments design doesn't match the wireframe given.

File: app/views/posts/index.html.erb (hunk 2 above)
Comment (danmainah) [reply app/views/posts/index.html.erb]: - [ ] @frnega as previously requested kindly fix the error as shown in the screenshot below, also ensure the page design matches the design given
image

File: app/views/users/show.html.erb (hunk 1 above)
Comment (danmainah) [reply app/views/users/show.html.erb]: - [ ] @frnega kindly ensure your user page shows only the recent posts from the user and not all of them as previously requested

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @frnega , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by frnega** [comment]
user
posts
postsgreater than 3

**Comment by frnega** [comment]
postload

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @frnega    :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**